README.md

```diff
@@ -73,10 +73,10 @@ An object is identified as a built-in using:
 
 #### `[[Builtin]]` internal slot
 
-All built-ins, with the exception of the intrinsic object `%ObjectPrototype%`,
-would have a `[[Builtin]]` internal slot with a string value identifying the
-name of the built-in. For instance, for the `%Math%` intrinsic object, the
-value of `[[Builtin]]` is `'Math'`.
+All built-ins, with the exception of the intrinsic objects `%ObjectPrototype%`,
+and `%Object%` have a `[[Builtin]]` internal slot with a string value
```

Comment:
I think `Builtin.typeof(Object) === 'function'` is going to be what most users would expect.

Question tho... what about prototype intrinsics like `%DatePrototype%`?

Should `Builtin.is(Object.getPrototypeOf(new Date()))` be `true` even if `Builtin.typeof(Object.getPrototypeOf(new Date()))` equals `object`?